docs/source/usage/cli.rst

```diff
@@ -170,11 +170,15 @@ Image
 
 Image-based provisioning can be accomplished through the use of `Metalsmith`_. It requires the image to be uploaded into OpenStack Glance. Once that's done, a non-admin can run the following:
 
-+----------------+---------------------------------------------------------------------------------------------------------------------+
-|                | **Actions**                                                                                                         |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
-| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --ssh-public-key <path-to-key>`` |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
++-------------------+---------------------------------------------------------------------------------------------------------------------+
+|                   | **Actions**                                                                                                         |
++-------------------+---------------------------------------------------------------------------------------------------------------------+
+| Provision Node    | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --candidate <node-uuid> --ssh-public-key <path-to-key>`` |
++-------------------+---------------------------------------------------------------------------------------------------------------------+
+| Remove Node       | ``metalsmith undeploy <node-uuid>``                                                                                 |
++----------- -------+---------------------------------------------------------------------------------------------------------------------+
+| Remove Deployment | ``metalsmith undeploy <deployment-uuid>``                                                                           |
```

Comment:
Where can we get the deployment-uuid?

Reply:
@DanNiESh deployment-uuid and allocation-uuid are same.

Reply:
I only see metalsmith undeploy <NODE UUID> in "metalsmith --help". I've never used undeploy <deployment-uuid> before. Have you tried it out?

Reply:
Yes. I have tried and tested it multiple times. You can also try it on the demo server.